technology/cassandra/src/main/java/com/github/mizool/technology/cassandra/ClusterProducer.java

```diff
@@ -77,6 +87,26 @@ private void registerJdk8TimeCodecs(Cluster cluster)
             .register(LocalTimeCodec.instance);
     }
 
+    private PoolingOptions getPoolingOptions()
+    {
+        PoolingOptions poolingOptions = new PoolingOptions();
+
+        int maxRequestsPerConnectionLocal = Integer.parseInt(System.getProperty(
```

Comment:
loading the default value makes this code unnecessarily complex. furthermore, making an effort to re-set the default feels nonsensical and like cargo culting.

we should either hard-code the default value in our code or skip the setter call if the respective system property wasn't set. I prefer the latter.